### PR TITLE
Max frame size

### PR DESCRIPTION
**Pull request by feiwan1**

### CODE REVIEW

File: doc/encoders.texi [hunk 1]
```diff
@@ -3630,6 +3630,10 @@ performance. This option doesn't work if driver doesn't implement vaSyncBuffer
 function. Please make sure there are enough hw_frames allocated if a large
 number of async_depth is used.
 
+@item max_frame_size
+Set the allowed max size in bytes for each frame. If the frame size exceeds
+the limitation, encoder will adjust the QP value to control the frame size.
```
Comment (xhaihao) [inline doc/encoders.texi:3635]: You added a check for CQP in the new version, so it'd be better to add a description here.

Comment (feiwan1) [reply doc/encoders.texi:3635]: Added. thanks

**Review by xhaihao — APPROVED** [review]
LGTM

**Comment by xhaihao** [comment]
See https://patchwork.ffmpeg.org/project/ffmpeg/list/?series=6462

**Pull request closed without merging**